gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Jun 25 17:16:27 CEST 2018
+#Fri Jun 05 11:29:26 CEST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```

Comment:
current Gradle version is 6.5. Do you deliberately keep 5.6.4?

Reply:
When I update it to the latest version I get the error: `Failed to resolve: com.squareup.okhttp3:okhttp:4.7.2` not sure why because it is the latest version available as you can check [here](https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp).

I would like to close this PR and move on for now, we can tackle this issue in another moment, what do you think @MyDogTom ?

Reply:
I agree.